src/cook-web/src/app/c/[[...id]]/Components/ChatUi/ChatUi.module.scss

```
@@ -171,10 +171,18 @@
     display: flex;
     flex: 1 1 0;
     height: calc(
-      calc(var(--app-viewport-block-unit, 1dvh) * 100) - 58px - 40px
+      calc(var(--app-viewport-block-unit, 1dvh) * 100) - var(
+          --mobile-chat-header-height,
+          44px
+        ) -
```

Stylelint (17.7.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)